portal-web/test/functional/com/liferay/portalweb/paths/pathlib/uielements/IFrame.path

```diff
@@ -62,7 +62,7 @@
 </tr>
 <tr>
 	<td>DIALOG_BOX</td>
-	<td>//div[contains(@class,'modal-header')]/h3</td>
+	<td>//div[contains(@class,'modal-header')]</td>
```

Comment:
This change fixes the path being used in the teardown of this file, also fixing the tests reported in [this]([LPS-184151](https://issues.liferay.com/browse/LPS-184151)) other ticket.